libc/benchmarks/gpu/LibcGpuBenchmark.h

```diff
@@ -146,10 +146,8 @@ template <typename T> class MathPerf {
       cpp::numeric_limits<StorageType>::max();
 
 public:
-  typedef T Func(T);
-
   template <size_t N = 1>
-  static uint64_t run_throughput_in_range(Func f, int min_exp, int max_exp) {
+  static uint64_t run_throughput_in_range(T f(T), int min_exp, int max_exp) {
```

Comment:
I'm guessing you needed the `f(T)` so overloading worked?

Reply:
Yeah, using the typedef meant it couldn't take in both types of functions